addons/account_analytic_default_hr_expense/models/hr_expense.py

```diff
@@ -9,8 +9,6 @@ class HrExpense(models.Model):
 
     @api.onchange('product_id', 'date', 'account_id')
     def _onchange_product_id(self):
-        res = super(HrExpense, self)._onchange_product_id()
```

Comment:
Is quite strange/error prone to remove the call to super. Why did you do that ?

Reply:
Because he removed the _onchange_product_id on the main hr_expense class.